projects/plugins/jetpack/extensions/blocks/blogroll/blogroll-item/blogroll-item.php

```diff
@@ -69,24 +70,34 @@ function load_assets( $attr, $content, $block ) {
 	if ( $is_following ) {
 		$subscribe_text            = esc_html__( 'Subscribed', 'jetpack' );
 		$disabled_subscribe_button = 'disabled';
+		$subscribe_button_class    = 'is-style-outline';
 	}
 
 	if ( empty( $icon ) ) {
 		$icon = 'https://s0.wp.com/i/webclip.png';
 	}
 
-	$buttons      = <<<HTML
-		<!-- wp:buttons -->
-		<div class="wp-block-buttons"><!-- wp:button {"className":"is-style-fill"} -->
-		<div class="wp-block-button is-style-fill"><button type="submit" name="blog_id" value="$id" class="wp-block-button__link wp-element-button">$submit_text</button></div>
+	$form_buttons      = <<<HTML
+		<!-- wp:button {"className":"is-style-fill"} -->
+		<div class="wp-block-button is-style-fill">
+			<button type="submit" name="blog_id" value="$id" class="wp-block-button__link wp-element-button">$submit_text</button>
+		</div>
 		<!-- /wp:button -->
 
 		<!-- wp:button {"className":"is-style-outline"} -->
-		<div class="wp-block-button is-style-outline"><button type="reset" class="wp-block-button__link wp-element-button jetpack-blogroll-item-cancel-button">$cancel_text</button></div>
-		<!-- /wp:button --></div>
-		<!-- /wp:buttons -->
+		<div class="wp-block-button is-style-outline">
+			<button type="reset" class="wp-block-button__link wp-element-button jetpack-blogroll-item-cancel-button">$cancel_text</button>
+		</div>
+HTML;
+	$form_buttons_html = do_blocks( $form_buttons );
+
+	$subscribe_button = <<<HTML
+		<!-- wp:button {"className":"$subscribe_button_class"} -->
+		<div class="wp-block-button is-style-fill jetpack-blogroll-item-subscribe-button $subscribe_button_class">
```

Comment:
We need to follow through with the dynamic class here
```suggestion
		<div class="wp-block-button $subscribe_button_class jetpack-blogroll-item-subscribe-button $subscribe_button_class">
```

Reply:
Oh yeah, we do, but I forgot to delete the hardcoded one.

Reply:
Pushed.